### PR TITLE
Add more instructions for installing OptiFine

### PR DESCRIPTION
**Pull request by Imzxhir**
Signed-off-by: Zxhir <98621617+Imzxhir@users.noreply.github.com>

### CODE REVIEW

File: src/wiki/getting-started/installing-optifine.md [hunk 1]
```diff
@@ -26,6 +26,13 @@ Now in Prism Launcher edit the instance you wish to install OptiFine on, open th
 
 OptiFine will now be installed as a jarmod in that instance so you can remove the installer and extracted jarmod.
 
+From OptiFine Version **H1_pre2** the instructions for installing OptiFine on PrismLauncher have changed. Make sure to have the version of optifine that you wish to install, installed through the vanilla launcher.
+
+1. Extract the MOD jar with the OptiFine installer
+2. Open it with any archiver
+4. Move all files from the folder called **notch** to the root folder and click **save** (By root folder, I mean the top level folder of the archive file)
```
Comment (DioEgizio) [inline src/wiki/getting-started/installing-optifine.md:33]:
```suggestion
3. Move all files from the folder called **notch** to the top folder of the archive file and click **save** 
```

File: src/wiki/getting-started/installing-optifine.md [hunk 2]
```diff
@@ -26,6 +26,13 @@ Now in Prism Launcher edit the instance you wish to install OptiFine on, open th
...
+5. Open Prism Launcher, edit the instance you wish to install OptiFine on, open the **Version** tab and click **Add to Minecraft.jar**, select the extracted / modfied OptiFine jarmod (the file ending in \_MOD.jar) and confirm.
```
Comment (DioEgizio) [inline src/wiki/getting-started/installing-optifine.md:34]:
```suggestion
4. Open Prism Launcher, edit the instance you wish to install OptiFine on, open the **Version** tab and click **Add to Minecraft.jar**, select the extracted / modfied OptiFine jarmod (the file ending in \_MOD.jar) and confirm.
```

File: src/wiki/getting-started/installing-optifine.md [hunk 3]
```diff
@@ -26,6 +26,13 @@ Now in Prism Launcher edit the instance you wish to install OptiFine on, open th
 
 OptiFine will now be installed as a jarmod in that instance so you can remove the installer and extracted jarmod.
 
+From OptiFine Version **H1_pre2** the instructions for installing OptiFine on PrismLauncher have changed. Make sure to have the version of optifine that you wish to install, installed through the vanilla launcher.
+
+1. Extract the MOD jar with the OptiFine installer
+2. Open it with any archiver
+3. Move all files from the folder called **notch** to the top folder of the archive file and click **save**
```
Comment (DioEgizio) [inline src/wiki/getting-started/installing-optifine.md:33]:
```suggestion
3. Move all files from the folder called **notch** to the top folder of the archive file and click **save**.
```

**Review by DioEgizio — COMMENTED** [review]
Remove this space

File: src/wiki/getting-started/installing-optifine.md [hunk 4]
```diff
@@ -26,6 +26,13 @@ Now in Prism Launcher edit the instance you wish to install OptiFine on, open th
 
 OptiFine will now be installed as a jarmod in that instance so you can remove the installer and extracted jarmod.
 
+From OptiFine Version **H1_pre2** the instructions for installing OptiFine on PrismLauncher have changed. Make sure to have the version of optifine that you wish to install, installed through the vanilla launcher.
```
Comment (DioEgizio) [inline src/wiki/getting-started/installing-optifine.md:29]:
```suggestion
From OptiFine Version **H1_pre2** the instructions for installing OptiFine on Prism Launcher have changed. Make sure to have the version of optifine that you wish to install, installed through the vanilla launcher.
```

File: src/wiki/getting-started/installing-optifine.md [hunk 5]
```diff
@@ -26,6 +26,13 @@ Now in Prism Launcher edit the instance you wish to install OptiFine on, open th
 
 OptiFine will now be installed as a jarmod in that instance so you can remove the installer and extracted jarmod.
 
+From OptiFine Version **H1_pre2** the instructions for installing OptiFine on Prism Launcher have changed. Make sure to have the version of optifine that you wish to install, installed through the vanilla launcher.
+
+1. Extract the MOD jar with the OptiFine installer
+2. Open it with any archiver
+3. Move all files from the folder called **notch** to the top folder of the archive file and click **save**.
+4. Open Prism Launcher, edit the instance you wish to install OptiFine on, open the **Version** tab and click **Add to Minecraft.jar**, select the extracted / modfied OptiFine jarmod (the file ending in \_MOD.jar) and confirm.
```
Comment (txtsd) [inline src/wiki/getting-started/installing-optifine.md:34]: modfied -> modified

Please run it through spellcheck next time

File: src/wiki/getting-started/installing-optifine.md [hunk 6]
```diff
@@ -26,6 +26,13 @@ Now in Prism Launcher edit the instance you wish to install OptiFine on, open th
 
 OptiFine will now be installed as a jarmod in that instance so you can remove the installer and extracted jarmod.
 
+From OptiFine Version **H1_pre2** the instructions for installing OptiFine on Prism Launcher have changed. Make sure to have the version of optifine that you wish to install, installed through the vanilla launcher.
```
Comment (DioEgizio) [inline src/wiki/getting-started/installing-optifine.md:29]:
```suggestion
From OptiFine Version **H1_pre2** the instructions for installing OptiFine on Prism Launcher have changed. Make sure to have the version of Minecraft that you wish to install OptiFine on installed through the vanilla launcher.
```

**Review by DioEgizio — APPROVED** [review]

**Pull request merged**